app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -62,10 +62,13 @@ def rrd_status
     'unknown'
   end
 
+  # If there was an attempt to check for a pending ep in the past, and no pending ep found, return false
   # If a pending_ep was detected in the past, returns true
-  # Depends on offramp_reason in form JSON, which is inserted by pending_eps? method
+  # If no evidence of checking for pending ep, perform the check
+  # Depends on offramp_reason and pending_eps in form JSON, which is inserted by pending_eps? method
   def had_pending_eps?
-    return true if form.dig('rrd_metadata', 'offramp_reason') == 'pending_ep'
+    return true if form.dig('rrd_metadata', 'pending_eps') == true
+    return false if form.dig('rrd_metadata', 'pending_eps') == false
```

Comment:
Opinion: this is slightly better to understand:
```suggestion
    return form.dig('rrd_metadata', 'pending_eps') if form.dig('rrd_metadata', 'pending_eps').present?
```
If there is a value of `pending_eps` then return that value. There would be a value if RRD called `pending_eps?` in the past.